Widescreen patch
Twin Castle, Emille's painting prison
Rose
Old: Disappears when the party crosses the bridge
New: No disappearing. Added movement and animation calls.
--- original
+++ modified
@@ -474,8 +474,44 @@
 mov 0xb, stor[24]
 mov 0x1, stor[25]
 gosub inl[:LABEL_16]
-call 106, 0x1
# Rotate and move
+mov 0xffffff42, inl[:WS_TARGET_X]
+mov 0, inl[:WS_TARGET_Y]
+mov 0x10e, inl[:WS_TARGET_Z]
+mov 10, inl[:WS_FRAMES]
+mov 1, inl[:WS_ANIMATION_0]
+gosub inl[:WS_ROTATE]
+mov 3, inl[:WS_SPEED]
+gosub inl[:WS_MOVE]
# Kneel
+mov 11, inl[:WS_ANIMATION_0]
+mov 0xffffffff, inl[:WS_ANIMATION_1]
+gosub inl[:WS_ANIMATION]
+mov 0xf, inl[:WS_WAIT_FLAG]
+mov 1, inl[:WS_WAIT_FLAG_MATCH]
+gosub inl[:WS_FLAG_WAIT]
# Stand up
+mov 12, inl[:WS_ANIMATION_0]
+mov 0xffffffff, inl[:WS_ANIMATION_1]
+gosub inl[:WS_ANIMATION]
# Rotate and move
+mov 0xffffff04, inl[:WS_TARGET_X]
+mov 0, inl[:WS_TARGET_Y]
+mov 0x60, inl[:WS_TARGET_Z]
+mov 10, inl[:WS_FRAMES]
+mov 1, inl[:WS_ANIMATION_0]
+gosub inl[:WS_ROTATE]
+mov 3, inl[:WS_SPEED]
+gosub inl[:WS_MOVE]
+call 97, 0
+mov 0x11, inl[:WS_WAIT_FLAG]
+mov 1, inl[:WS_WAIT_FLAG_MATCH]
+gosub inl[:WS_FLAG_WAIT]
+mov 20, inl[:WS_FRAMES]
+wait inl[:WS_FRAMES]
# Face Dart's direction
+call 102, var[64][0], inl[:WS_TARGET_X], inl[:WS_TARGET_Y], inl[:WS_TARGET_Z]
+mov 6, inl[:WS_FRAMES]
+mov 0, inl[:WS_ANIMATION_0]
+gosub inl[:WS_ROTATE]
# Animation freeze
+call 99, 1
 return
+#include ../../../../patches/libs/widescreen.txt
 LABEL_54:
 rel :PTR_13c4_0
 rel :PTR_13c4_1
